gems/pending/fs/ext4/superblock.rb

```diff
@@ -223,6 +223,14 @@ def inodeSize
       isDynamic? ? @sb['inode_size'] : INODE_SIZE
     end
 
+    def isEnabled64Bit?
+      gotBit?(@sb['incompat_flags'], ICF_64BIT)
+    end
+
+    def groupDescriptorSize
+      isEnabled64Bit? ? @sb['group_desc_size'] : GDE_SIZE 
```

Comment:
No, it's will be 0 if isEnabled64Bit? is false, so we have to check the flag condition